src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferEvidenceTonnageViewModelMap.cs

```diff
@@ -76,11 +76,11 @@ public TransferEvidenceTonnageViewModel Map(TransferEvidenceNotesViewModelMapTra
                         string reusedTonnage = null;
                         if (source.TransferAllTonnage)
```

Comment:
Same for this either unit test is currently wrong or we need to create one...

Reply:
Updated the unit test, I believe should be covered now